### ISSUE TITLE
Add alt text to all images

### ISSUE THREAD

**Issue by spydon** [flame-engine/flame#2913]
# What could be improved
We should have alt text for all images

# Why should this be improved
 To improve accessibility.

# Any risks?
No

# More information
Swap this to true:
https://github.com/flame-engine/flame/blob/main/.markdownlint.yaml#L204

And you'll get this:
```
packages/flame_audio/README.md:13:63 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_audio/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_audio/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_audio/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_bloc/README.md:13:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_bloc/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_bloc/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_bloc/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_fire_atlas/README.md:13:68 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_fire_atlas/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_fire_atlas/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_fire_atlas/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_forge2d/README.md:13:65 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_forge2d/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_forge2d/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_forge2d/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_isolate/README.md:13:65 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_isolate/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_isolate/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_isolate/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_jenny/README.md:13:63 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_jenny/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_jenny/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_jenny/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_lint/README.md:13:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_lint/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_lint/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_lint/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_lottie/README.md:13:64 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_lottie/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_lottie/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_lottie/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_markdown/README.md:13:66 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_markdown/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_markdown/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_markdown/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_network_assets/README.md:16:57 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_noise/README.md:13:63 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_noise/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_noise/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_noise/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_oxygen/README.md:13:64 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_oxygen/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_oxygen/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_oxygen/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_rive/README.md:13:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_rive/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_rive/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_rive/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_splash_screen/README.md:13:71 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_splash_screen/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_splash_screen/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_splash_screen/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_splash_screen/README.md:20:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_svg/README.md:13:61 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_svg/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_svg/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_svg/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_test/README.md:13:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_test/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_test/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_test/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_tiled/README.md:13:63 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_tiled/README.md:14:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_tiled/README.md:15:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame_tiled/README.md:16:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame/README.md:20:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame/README.md:21:108 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame/README.md:22:56 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame/README.md:23:62 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame/README.md:124:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame/README.md:130:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/flame/README.md:139:3 MD045/no-alt-text Images should have alternate text (alt text)
README.md:20:56 MD045/no-alt-text Images should have alternate text (alt text)
README.md:21:108 MD045/no-alt-text Images should have alternate text (alt text)
README.md:22:56 MD045/no-alt-text Images should have alternate text (alt text)
README.md:23:62 MD045/no-alt-text Images should have alternate text (alt text)
README.md:124:3 MD045/no-alt-text Images should have alternate text (alt text)
README.md:130:3 MD045/no-alt-text Images should have alternate text (alt text)
README.md:139:3 MD045/no-alt-text Images should have alternate text (alt text)
```

**Comment by spydon** [comment]
Should be completed in #2895
